meta-iot2000-example/recipes-tools/switchmode/files/switchmode.c

```diff
@@ -42,13 +41,13 @@ int main(int argc, char *argv[])
 
 	if (argc != 3) {
 		print_usage(argv[0]);
-		return -EINVAL;
+		return 1;
```

Comment:
I'm not aware of a convention here but I'd like to be able to differentiate the two failure cases based on the exit code, e.g., in a script. As said, nitpicking... It's up to your taste.